_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -713,7 +720,41 @@ public static function confirm_safe_mode() {
 				)
 			);
 		}
-		return new WP_Error( 'error_confirming_safe_mode', esc_html__( 'Jetpack could not confirm safe mode.', 'jetpack' ), array( 'status' => 500 ) );
+		return new WP_Error(
+			'error_setting_jetpack_safe_mode',
+			esc_html__( 'Could not confirm safe mode.', 'jetpack' ), array( 'status' => 500 )
+		);
```

Comment:
On all of these errors, the `array( 'status' => $x )` should be on the next line I believe.